### PR TITLE
separate s3 mgrt tests from other cli tests

### PR DESCRIPTION
**Pull request by Kishan-Dhakan**

### CODE REVIEW

File: .github/workflows/manual_system_tests.yml [hunk 1]
```diff
@@ -260,7 +265,7 @@ jobs:
 
     - name: "Run System tests"
       if: ${{ env.SKIP_TESTS != 'true' }}
-      uses: 0chain/actions/run-system-tests@master
+      uses: 0chain/actions/run-system-tests@enable-s3mgrt-only-tests
```
Comment (stewartie4) [inline .github/workflows/manual_system_tests.yml:268]: revert

File: .github/workflows/manual_system_tests.yml [hunk 2]
```diff
@@ -233,7 +238,7 @@ jobs:
 
     - name: "Deploy 0Chain"
       if: ${{ env.SKIP_TESTS != 'true' }}
-      uses: 0chain/actions/deploy-0chain@master
+      uses: 0chain/actions/deploy-0chain@enable-s3mgrt-only-tests
```
Comment (stewartie4) [inline .github/workflows/manual_system_tests.yml:241]: revert

File: .github/workflows/manual_system_tests.yml [hunk 3]
```diff
@@ -213,7 +218,7 @@ jobs:
         svc_account_secret: ${{ secrets.SVC_ACCOUNT_SECRET }}
 
     - name: "Set PR status as pending"
-      uses: 0chain/actions/notify-pull-request@master
+      uses: 0chain/actions/notify-pull-request@enable-s3mgrt-only-tests
```
Comment (stewartie4) [inline .github/workflows/manual_system_tests.yml:221]: revert

File: .github/workflows/manual_system_tests.yml [hunk 4]
```diff
@@ -201,7 +206,7 @@ jobs:
 
     - name: "Get docker images for feature branches"
       if: ${{ env.SKIP_TESTS != 'true' }}
-      uses: 0chain/actions/get-latest-image-for-branch@master
+      uses: 0chain/actions/get-latest-image-for-branch@enable-s3mgrt-only-tests
```
Comment (stewartie4) [inline .github/workflows/manual_system_tests.yml:209]: revert

File: run-system-tests/action.yml [hunk 5]
```diff
@@ -434,6 +446,40 @@ runs:
...
+        export HOME="/root"
+
+        GOPATH=$(echo $(go env GOPATH))
+
+        cd tests/cli_tests
+        system_tests_exit_code=0
+        echo "Sequential tests are currently running.... It will take a few minutes for initial test output to appear..."
+        echo
+        ${GOPATH}/bin/gotestsum --jsonfile test-output.json --hide-summary=output --format testname ${{ env.SHOULD_RETRY_FAILURES }} --raw-command -- ../../S3CLI_TEST_RUNNER_COMMAND.sh || system_tests_exit_code=$?
+        cat test-output.json | ${GOPATH}/bin/go-test-report --groupSize 1 --output ../../${{ env.BRANCH_DIR }}/${GITHUB_SHA}/cli/index.html --title "0Chain S3 CLI System tests [${{ env.BRANCH_DIR }}/${GITHUB_SHA:0:8}] ran against [${{ env.NETWORK_URL }}] at [${{ env.TEST_TIME }}]"
```
Comment (stewartie4) [inline run-system-tests/action.yml:468]: This looks like it's overriding the CLI tests report page

**Review by stewartie4 — CHANGES_REQUESTED** [review]

File: run-system-tests/action.yml (hunk 5 above)
Comment (Kishan-Dhakan) [reply run-system-tests/action.yml]: Fixed these, now it goes to /s3cli/ dir
image

File: .github/workflows/manual_system_tests.yml [hunk 6]
```diff
@@ -292,7 +298,7 @@ jobs:
 
     - name: "Set PR status as ${{ job.status }}"
       if: ${{ (success() || failure() || cancelled()) && inputs.test_file_filter == '' }}
-      uses: 0chain/actions/notify-pull-request@master
+      uses: 0chain/actions/notify-pull-request@enable-s3mgrt-only-tests
```
[the remark this answers is not in the document]
Comment (Kishan-Dhakan) [reply .github/workflows/manual_system_tests.yml]: resolved

File: .github/workflows/manual_system_tests.yml (hunk 2 above)
Comment (Kishan-Dhakan) [reply .github/workflows/manual_system_tests.yml]: resolved

File: .github/workflows/manual_system_tests.yml (hunk 4 above)
Comment (Kishan-Dhakan) [reply .github/workflows/manual_system_tests.yml]: resolved

File: .github/workflows/manual_system_tests.yml (hunk 1 above)
Comment (Kishan-Dhakan) [reply .github/workflows/manual_system_tests.yml]: resolved

File: deploy-0chain/action.yml [hunk 7]
```diff
@@ -102,7 +102,7 @@ runs:
     - name: "Setup Config"
       shell: 'script --return --quiet --command "bash {0}"'
       run: |
-        echo "ACTIONS_BRANCH=$(cd ../../_actions/0chain/actions/ && find . -name .github | grep -o -P '(?<=./).*(?=/.github)')" >> $GITHUB_ENV
+        echo "ACTIONS_BRANCH=enable-s3mgrt-only-tests" >> $GITHUB_ENV
```
[the remark this answers is not in the document]
Comment (Kishan-Dhakan) [reply deploy-0chain/action.yml]: resolved

**Pull request merged**